App/src/views/NewRequests.js

```diff
@@ -0,0 +1,127 @@
+/** @jsx jsx */
+
+import React, { useState, useEffect } from "react";
+import { jsx } from "@emotion/core";
+import { Redirect } from "@reach/router";
+import Nabvar from "../components/Nabvar";
+import { requestsFetch, requestAdmin } from "../services/request";
+import RequestModal from "../components/RequestModal";
+import { useUser } from "../contexts/user";
+import Request from "../components/Request";
+import { users } from "../services/user";
+
+const Requests = () => {
+  const [requests, setRequests] = useState([]);
+  const [requestsAdmin, setRequestsAdmin] = useState([]);
+  const [frontdesks, setFrontdesks] = useState([]);
+
+  const [id, setId] = useState(0);
+  const user = useUser();
+
+  useEffect(() => {
+    async function fetchData() {
+      const response = await requestsFetch();
+      setRequests(response);
+    }
+    fetchData();
+  }, []);
+
+  useEffect(() => {
+    async function fetchData() {
+      const response = await requestAdmin();
+      setRequestsAdmin(response);
+    }
+    fetchData();
+  }, []);
+
+  useEffect(() => {
+    async function fetchData() {
+      const response = await users();
+      setFrontdesks(response);
+    }
+    fetchData();
+  }, []);
+
+  function findName(requester_id) {
+    return frontdesks.find(frontdesk => frontdesk.id === requester_id).name;
+  }
+
+  if (!user) return <Redirect to="login" noThrow />;
+  if (frontdesks.length === 0) return "Cargando...";
+  if (!requests.length === 0) return "Cargando...";
+
+  return (
+    <div>
+      <Nabvar />
+      <div
+        name="request"
+        css={{
+          flexShrink: "0",
+          display: "flex",
+          width: "100%"
+        }}
+      >
+        <div
+          name="requestList"
+          css={{
+            display: "block",
+            width: "350px"
+          }}
+        >
+          <h2
+            css={{
+              fontSize: "25px",
+              color: "#35469c",
+              paddingLeft: "16px",
+              fontWeight: 400
+            }}
+          >
+            Requests
+          </h2>
+          <ul name="list" css={{ padding: 0, margin: 0 }}>
+            {requests
```

Comment:
Remember to merge your `requests` and `requestsAdmin` here to support them both.
```suggestion
            {requests.concat(requestsAdmin)
```

Reply:
Thank you, Now Admin can visualize the request list